package.json

```diff
@@ -11,6 +11,7 @@
     "@testing-library/user-event": "^12.8.3",
     "dotenv": "^16.0.3",
     "firebase": "^8.6.3",
+    "mdb-react-ui-kit": "^4.2.0",
```

Comment:
dependencies for react are getting broken!
Do not use ui-kit for now. It will broke other UI as well!